packages/ui/src/ui-component/switch/Switch.jsx

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types'
 import { FormControl, Switch, Typography } from '@mui/material'
 
 export const SwitchInput = ({ label, value, onChange, disabled = false }) => {
+    //@ts-ignore https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing
```

Comment:
instead of `ts-ignore`, try this fix - https://github.com/FlowiseAI/Flowise/pull/1946/files#diff-d034f73b36aff30fea6c0600a763fefe400056697b0470ab1579c95b48d42047R6

Reply:
ts-ignore removed